totalsize/total.py

```diff
@@ -430,6 +431,10 @@ def cli():
             csv_path = Path(csv_file)
             write_to_csv(csv_path, gen_csv_rows([MOCK_ENTRY]))
             csv_path.unlink()
+        
+        if args.cookies:
+            global YTDL_OPTS
+            YTDL_OPTS['cookiefile'] = args.cookies
```

Comment:
`YTDL_OPTS` is a constant, so you're not supposed to modify it. Instead, you could add a `cookies` param to the initializer of `Playlist`, and then have a new (local) variable in the initializer, with the contents of `YTDL_OPTS` plus your cookies stuff. You'd have to modify the creation of the `playlist` object in `cli` accordingly of course.

Reply:
Ok, I've tried fixing it.